lua/autorun/sliding_ability.lua

```diff
@@ -173,20 +169,27 @@ local function IsSliding(ply)
     return ply:GetNWBool("SlidingAbilityIsSliding", false)
 end
 
+local kaitCvar = GetConVar("kait_movement_enabled")
+
 local function CheckCrouching(ply)
     -- Compatibility issue with Movement - Reworked partially fixes by disabling crouch check
-    if GetConVar "kait_movement_enabled" then return true end
+    if kaitCvar then return true end
     return ply:Crouching()
 end
 
+local savavCvar = GetConVar("savav_parkour_Enable")
+local exosuitCvar = GetConVar("sv_sliding_enabled")
+
 hook.Add("SetupMove", "SlidingAbility_CheckSliding", function(ply, mv)
+    if not CheckCrouching(ply) then return end
```

Comment:
I was really unsure if checking this one first is fine because I was afraid of making more prediction errors.  WIth a quick test using `net_fakelag 300` on my local SRCDS, it seems no much difference about the prediction errors with this change, so it's good to go.

Reply:
Yeah, after months of testing this, I'm feeling pretty confident that the issue isn't made any worse by this change.